mod.ts

```diff
@@ -43,18 +56,44 @@ class Renderer extends Marked.Renderer {
   }
 }
 
+/** Convert inline and block math to katex */
+function mathify(markdown: string) {
+  // Deal with inline math
+  const inlineMath = /\$(\S.+\S)\$/;
+  let match;
+  while ((match = inlineMath.exec(markdown)) !== null) {
+    markdown = markdown.replace(
+      inlineMath,
+      katex.renderToString(match[1], { displayMode: false }),
+    );
+  }
+
+  // Deal with block math
+  const blockMath = /\$\$\s([\s\S]+)\s\$\$/;
```

Comment:
this regex is odd

Reply:
How so? I'm checking for double dollar signs + at least one whitespace character + any character + at least one whitespace character + double dollar signs. `.+` doesn't work because it doesn't match newlines but if you prefer I could use `[.\n]+`